src/components/CallView/LocalMediaControls.vue

```diff
@@ -122,6 +127,25 @@ export default {
 
 	computed: {
 
+		conversation() {
+			if (this.$store.getters.conversations[this.token]) {
+				return this.$store.getters.conversations[this.token]
+			}
+
+			return {
+				sessionId: '0',
+				participantType: this.$store.getters.getUserId() !== null ? PARTICIPANT.TYPE.USER : PARTICIPANT.TYPE.GUEST,
+				objectType: null,
+			}
+		},
+
+		isScreensharingAllowed() {
+			return this.conversation.participantType === PARTICIPANT.TYPE.OWNER
```

Comment:
shouldn't we have a flag somewhere that says whether this call (or room) allows anyone to to share screens instead of just moderators ?

Reply:
ok, it was noted in the PR comment already